src/Tribe/Aggregator/Record/List_Table.php

```diff
@@ -520,6 +520,16 @@ public function column_source( $post ) {
 			$html[] = '</dl></div>';
 		}
 
+		/**
+		 * Customize the Events > Import > History > Source column HTML.
+		 *
+		 * @since TBD
+		 *
+		 * @param array                                       $html   List of HTML details.
+		 * @param Tribe__Events__Aggregator__Record__Abstract $record The record object.
```

Comment:
Pass `use Tribe__Events__Aggregator__Record__Abstract as Record_Abstract;` to make docblock smaller.

Reply:
I usually do that but if I don't use the class in PHP then PhpStorm will remove it on cleanup/auto-format of the file -- docblocks don't count in their checks there.

I can do that here though.